test/integration/api-create.test.js

```diff
@@ -993,3 +991,433 @@ test('run external network (docker run --label triton.network.public=)',
         });
     });
 });
+
+/*
+ * Tests for `docker run --label trtiton.network.public_ipv4`
```

Comment:
Here is the typo I mentioned.
```suggestion
 * Tests for `docker run --label triton.network.public_ipv4`
```

Reply:
Sorry, this somehow got lost in my initial review :confused: